src/content/peerconnection/constraints/js/main.js

```diff
@@ -249,20 +249,31 @@ setInterval(function() {
 
       // search for the candidate pair
       results.forEach(function(report) {
-        if (report.type === 'candidatepair' && report.selected ||
-            report.type === 'googCandidatePair' &&
-            report.googActiveConnection === 'true') {
-          activeCandidatePair = report;
+        if (report.type === 'transport') { 
+          activeCandidatePair = results.get(report.selectedCandidatePairId); 
         }
       });
+      if (!activeCandidatePair) {
+        results.forEach(function(report) {
+          if (report.type === 'candidate-pair' && report.selected ||
```

Comment:
comments added. I think it is easier to read and understand what is the legacy fallback when the forEach's are separated